### DISCUSSION TITLE
MiraiTravel 接收到信息然后发送不出去指令

### DISCUSSION THREAD

**Discussion by yixianbaowang** [MR-XieXuan/MiraiTravel#7] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/125167113/218351655-b852b9b3-f1fe-4857-a33e-ddd0c2d457e1.png)

**Comment by MR-XieXuan** [comment]
@yixianbaowang 看看你的脚本

**Comment by MR-XieXuan** [comment]
如果发不出消息的话你需要检查 : 是否调用成功了 MiraiApi 。
具体查看MiraiApiHTTP的日志.

**Comment by MR-XieXuan** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/95032548/218387071-6d3602de-377e-468e-80d4-89fe1c2219fc.png)
c查出问题是没有正确配置 AUTHORIZATION ,

**Comment by MR-XieXuan** [comment]
![image](https://user-images.githubusercontent.com/95032548/218387274-da9c1937-c1fa-4e36-b1cf-eb833a4b4bd4.png)
在webhook的日志中出现 AUTHORIZATION出现严重错误
